### PR TITLE
Create moderation summaries README.md

### PR DESCRIPTION
**Pull request by GeorgiaHCA**
This PR adds a README to contextualise and provide extra info for the moderation data. It would be great if reviewers could check if this is readable and clear for someone not already that familiar with the project, as well as spelling and links. Thanks!

### CODE REVIEW

File: community-recommendations/moderation-co-creation/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* *F* = Fujitsu
+
+#### Specialist Role on Project
+
+* R = Researcher
+
+#### Connection to Autism
+
+* **A** = Autistic
+* **P** = Parent of someone autistic
```
Comment (amol-prabhu) [inline community-recommendations/moderation-co-creation/README.md:31]: Perhaps someone with autism or an autistic person?

File: community-recommendations/moderation-co-creation/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+We have also taken out references to specific locations, institutions, and organisations which might identify an individual. 
+For the purposes of creating a public dataset which is autism-voice first, we have edited out comments which contain negative emotional responses, struggles, or judgements that non-autistic people have about autistic people, where the emphasis is on non-autistic people’s experiences (e.g. personal reactions to autistic people’s behaviours). 
+We have kept comments about non-autistic people’s responses to autistic people, when the emphasis is on autistic people’s experiences (e.g. comments about dealing with neurotypical prejudice). 
+
+Feedback from the AutSPACEs community, including from autistic people, autistic parents of autistic people, and parents of autistic people, has been prdominantly that there should be an emphasis on autistic people’s experiences first and foremost, and that having too much focus on non-autistic views of autistic people can be damaging to autistic people, and work against creating a safe space for autistic voices to be heard.
```
Comment (amol-prabhu) [inline community-recommendations/moderation-co-creation/README.md:50]: predominantly*

**Review by amol-prabhu — COMMENTED** [review]
Looks great to me! Have just noticed a couple of typos and perhaps a rephrasing, other than that it's fab!

**Comment by amol-prabhu** [comment]
AP Review

**Comment by GeorgiaHCA** [comment]
Thanks @amol-prabhu ! Made those edits. If you could approve the PR I'll merge it

**Review by amol-prabhu — APPROVED** [review]

**Comment by amol-prabhu** [comment]
Done! ☺

**Pull request merged**